src/calendar_duration.rs

```diff
@@ -268,9 +270,31 @@ impl CalendarDuration {
     }
 }
 
+impl From<Days> for CalendarDuration {
```

Comment:
Let's skip the `From<Days>` and `From<Months>` impl pending decisions about whether we'll deprecate those.